### ISSUE TITLE
unicodeToImage class does not match PNG filename

### ISSUE THREAD

**Issue by dbrgn** [Ranks/emojione#484]
When using spritemaps:

```javascript
> emojione.unicodeToImage('☹️')
"<span class="e1 e1-people _2639-fe0f" title=":frowning2:">☹️</span>"
```

The codepoint used in the class name is `2639-fe0f`, but the filename of the corresponding emoji PNG from the download archive is just `2639.png`. When generating spritemaps based on the name, the two don't match.

It would be good if the two would match. Otherwise they have to be mapped manually, which is probably not the intent of this library.

(PS: Did you change the class name prefix from `emojione-` to `_` on purpose?)

**Comment by caseyahenson** [comment]
@dbrgn thanks for pointing this out! I see the issue here and I'll get it fixed soon. We made a number of changes to the sprite structure with 3.0, the naming pattern you show above is not quite accurate but the new class specification is intentional. One class specifies the category and size, e.g. `emojione-32-people` and another specifies the file name, e.g. `_2639`.

On a related note, we'll also be adding the option to set the sprite size as either 32 or 64, which will be available soon.

**Comment by dbrgn** [comment]
> the naming pattern you show above is not quite accurate

Yes, sorry, I renamed emojione and emojione-32 to e1 to make the spritemap CSS file smaller.

> On a related note, we'll also be adding the option to set the sprite size as either 32 or 64, which will be available soon.

I generated the spritemap myself by tweaking the config files from the v2 version, but if you're officially supporting them with v3 (for the 32x32 version), then that's even better :)

**Comment by dbrgn** [comment]
Any ETA on this? :)

**Comment by caseyahenson** [comment]
The patch for this should be released today.

**Comment by caseyahenson** [comment]
@dbrgn this is fixed in 3.0.2, released today.
